### PR TITLE
BaSP-TG-63: Created select dropdown

### PR DESCRIPTION
**Pull request by FiolAxel**
image
To use: 

1. import SelectDropdown from '../Shared/SelectDropdown';
```
2. const options = [
    { value: 10, label: 'Select 1' },
    { value: 10, label: 'Select 2' }
  ];
3. return (
    <section className={styles.container}>
      <h2>Employees</h2>
      <div>
        <List list={list} deleteItem={deleteItem} setList={setList} />
      </div>
      <a className={styles.button} href="/employees/form">
        Add New Employee
      </a>
      <SelectDropdown options={options} />
    </section>
  );
};
```

### CODE REVIEW

**Review by Facund0Fernandez — DISMISSED** [review]

**Review by laura-brussa12 — APPROVED** [review]

File: src/Components/Shared/SelectDropdown/sd.module.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.select {
+  width: 7%;
+  height: 30px;
+  margin-top: 7px;
+  padding: 0 12px;
+  border: 1px solid #000;
```
Comment (GuidoCerioni) [inline src/Components/Shared/SelectDropdown/sd.module.css:6]: Here, border color should be #373867

File: src/Components/Shared/SelectDropdown/index.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      onChange={props.onChange}
+      required={props.required}
+      pattern={props.pattern}
+      className={styles.select}
+    >
+      <option value="" disabled>
+        Select one
+      </option>
+      {props.options.map((option) => (
+        <option key={option.value} value={option.value}>
```
Comment (GuidoCerioni) [inline src/Components/Shared/SelectDropdown/index.js:18]: Here, keys must be unique, and values are the same in options

**Review by GuidoCerioni — CHANGES_REQUESTED** [review]
Hi Alex, check the comments!

**Review by Facund0Fernandez — APPROVED** [review]

**Review by laura-brussa12 — APPROVED** [review]

**Review by GuidoCerioni — APPROVED** [review]
Nice!!!

**Pull request merged**